src/System/Taffybar/Widget/Brightness.hs

```diff
@@ -0,0 +1,13 @@
+module System.Taffybar.Widget.Brightness (brightnessBarNew) where
+
+import Control.Concurrent.Chan (dupChan)
+import Graphics.UI.Gtk (Widget)
+import System.Taffybar.Widget.Generic.VerticalBar (BarConfig)
+import System.Taffybar.Widget.Generic.PollingBar (verticalBarFromChannel)
+import System.Taffybar.Information.Brightness (readBrightnessValues)
+
+brightnessBarNew :: BarConfig -> IO Widget
```

Comment:
All widgets need to either be in TaffyIO or MonadIO m.

I don't think you would be able to add this to any of start, end or center widgets as is, right?